packages/platform-sdk-profiles/src/environment/env.ts

```diff
@@ -136,13 +137,7 @@ export class Environment {
 			profiles = {};
 		}
 
-		const validated = schema.validateSync(
-			{ data, profiles },
-			{
-				strict: true,
-			},
-		);
-
+		const validated = validator.validate({ data, profiles }, schema, { strict: true });
 		if (validator.fails()) {
```

Comment:
@faustbrian also moved this validation into `Validator` as the `validation as `validator.fails()` was always false previously with `schema.validateSync`